middle_end/flambda2/to_cmm/to_cmm_expr.ml

```diff
@@ -774,8 +753,38 @@ and apply_expr env res apply =
           Env.set_inlined_debuginfo env handler_body_inlined_debuginfo
         in
         expr env res body
-      | _ :: _ -> unsupported ())
-    | Jump _ -> unsupported ())
+      | params ->
+        let wrap, env, res =
+          Env.flush_delayed_lets ~mode:Branching_point env res
+        in
+        let env, cmm_params =
+          Env.create_bound_parameters env (List.map Bound_parameter.var params)
+        in
+        let label = Lambda.next_raise_count () in
```

Comment:
We should add a CR here to note to switch to a proper way to destructure the tuple returned by the call when we introduce one in `cmm`. That would also help removing some flushing as well.